### PR TITLE
[T2] confluent for right output folder

### PR DESCRIPTION
**Pull request by msyyc**
the output folder is not right for python, better to fix ASAP.

### CODE REVIEW

**Review by alluri02 — APPROVED** [review]

**Review by ArcturusZhang — APPROVED** [review]

**Comment by ArcturusZhang** [comment]
I cannot merge this PR since one of the required CI check is failing. @msyyc @akning-ms could you please help merge this PR when this is ready?

**Comment by msyyc** [comment]
> I cannot merge this PR since one of the required CI check is failing. @msyyc Yuchao Yan FTE @akning-ms Arthur Ning FTE could you please help merge this PR when this is ready?

I' sure it doesn't have any bad effect

**Pull request closed without merging**
